news/2022-01-12-new-featured-artist-haywyre.md

```diff
@@ -0,0 +1,57 @@
+
+This grammy-nominated producer and pianist has toured with the likes of *Zedd* and *GRiZ* thanks to the distinct vision he puts forth with each of his electronic tunes. Inspired by creators like our very own [KOAN Sound](https://osu.ppy.sh/beatmaps/artists/49) and various classical composers/pianists, **Haywyre**'s one-of-a-kind sound is surprisingly well suited for the world of rhythm gaming.
+
+**13** tracks are on the table today, each of which has a beatmap template ready to go from [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230)!
+
+Curious where the hype behind **Haywyre** comes from? His presence on osu! started with a fairly popular marathon of [Everchanging](https://osu.ppy.sh/beatmapsets/461903), later expanding even more through competition on [Insight](https://osu.ppy.sh/beatmapsets/848400)'s nearly inhuman jumps. See exactly what that means by watching [Vaxei](https://osu.ppy.sh/users/4787150) SS it below, then explore the other songs:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/230/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Insight
+
+Play through [the map above](https://osu.ppy.sh/beatmapsets/848400) hosted by [Twigglykun](https://osu.ppy.sh/users/9126943)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/230/Songs/Haywyre%20-%20Insight.mp3" type="audio/mpeg">
+</audio>
+
+### Everchanging
+
+Check out [the map that introduced **Haywyre** to osu!](https://osu.ppy.sh/beatmapsets/461903) hosted by [Beige](https://osu.ppy.sh/users/685229) (previously known as **W h i t e**)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/230/Songs/Haywyre%20-%20Everchanging.mp3" type="audio/mpeg">
+</audio>
+
+### Dichotomy (Soft Mix)
+
+If you're tired of circles, rectangles await you on [this osu!mania beatmap](https://osu.ppy.sh/beatmapsets/631054) hosted by [BilliumMoto](https://osu.ppy.sh/users/3862471)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/230/Two%20Fold%20Pt.%201/Haywyre%20-%20Dichotomy%20%28Soft%20Mix%29.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+If you're sitting in front of your computer, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) is waiting for you to choose a song and open its pre-timed beatmap template in the editor. If you're reading this on your phone, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) encourages you to have a good day and stay hydrated. Yes, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) is sentient.
+
+Fun fact: **Haywyre** was internally used as a mockup example for Featured Artist pages many years ago! Fun fact \#2: you can be notified about our next Featured Artist as soon as it drops from the [@osugame Twitter](https://twitter.com/osugame).
```

Comment:
2 Fun facts in same line?

Reply:
if he's daring, some day he'll do three!

this is fine otherwise, is a literary device